app/server/sendEndringAction.server.ts

```diff
@@ -2,16 +2,27 @@ import { sendEndringsmelding } from '~/server/sendEndringsmelding.server';
 import { getSession } from '~/sessions';
 import { IEndringsmelding } from '~/typer/endringsmelding';
 import { EStatusKode, IPostResponse } from '~/typer/response';
+import { EYtelse } from '~/typer/ytelse';
 
-export default async function sendEndringAction(request: Request) {
+export default async function sendEndringAction(
+  request: Request,
+  ytelse: EYtelse,
+) {
   const formData = await request.formData();
   const endringsmeldingTekst = formData.get('endringsmelding') as string;
   const endringsmelding: IEndringsmelding = {
     tekst: endringsmeldingTekst,
     dokumenter: [],
   };
+  let ytelseSti: string = '';
+  if (ytelse === EYtelse.BARNETRYGD) {
+    ytelseSti = 'ba';
+  } else if (ytelse === EYtelse.KONTANTSTØTTE) {
+    ytelseSti = 'ks';
+  }
```

Comment:
Denne funksjonen kan være en util-funksjon med switch case på EYtelse. Den vil være lik `hentPathForYtelse()` men kan kanskje være greit å ha to ulike funksjoner, en for frontend og en for backend, tilfelle de ikke alltid er like.